fsharp-backend/src/BackendOnlyStdLib/Init.fs

```diff
@@ -1,9 +1,12 @@
 module BackendOnlyStdLib.Init
 
-// Initialize BackendOnlyStdLib
-
 open Prelude
 
+// WHATISTHIS why do we have these when they're empty?
+
```

Comment:
There was something here but it wasn't needed anymore. You're right, we should just remove this.